README.md

```diff
@@ -1 +1,204 @@
-# REACT_CAPSTONE
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+
+  <h1><b>Web Metrix currency Exchange</b></h1>
+
+
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Web Metrix currency Exchange<a name="about-project"></a>
+
+ I developed a Currency conversion App using React,Redux  and APIs. This project aims to create a simple and intuitive interface that allows users to search their top currency even cryptocurrencies for conversion rate.👌 Builds the app for production to the build folder.
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/React/">React</a></li>
+    <li><a href="https://www.w3schools.com/Redux/">Redux</a></li>
+  </ul>
+</details>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+<!-- LIVE Video -->
+
+## 🎥 Live Video <a name="live-video"></a>
+
+- [Live video Link](https://www.loom.com/share/459edb0dcea34e6d92dd284e3b22c6c6?sid=f38439e3-87f9-4c98-ad3a-776de6fee35a)
+
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2:  the video highlights a piece of the code the student finds attractive or is proud of making but in your video, you didn't highlight the code which is proud of making so kindly re-create your video and also add the piece of code is proud of making :slightly_smiling_face: